src/protagonist/API.Tests/Integration/ModifyAssetTests.cs

```diff
@@ -414,6 +414,49 @@ public async Task Put_NewImageAsset_FailsToCreateAsset_WhenMediaTypeAndFamilyNot
         response.StatusCode.Should().Be(HttpStatusCode.BadRequest);
     }
     
+    [Fact]
+    public async Task Put_NewImageAsset_FailsToCreateAsset_WhenMatchingDefaultDeliveryChannelsAreInvalid()
+    {
+        // arrange
+        const int customerId = 9901;
+        var assetId = new AssetId(customerId, 1, nameof(Put_NewImageAsset_FailsToCreateAsset_WhenMatchingDefaultDeliveryChannelsAreInvalid));
+        var hydraImageBody = $@"{{
+          ""@type"": ""Image"",
+          ""origin"": ""https://example.org/{assetId.Asset}.tiff"",
+          ""mediaType"": ""application/tiff""
+        }}";
+
+        await dbContext.Spaces.AddTestSpace(customerId, 1);
+        
+        dbContext.DefaultDeliveryChannels.Add(new DLCS.Model.DeliveryChannels.DefaultDeliveryChannel()
+        {
+            Customer = customerId,
+            MediaType = "application/*",
+            DeliveryChannelPolicyId = KnownDeliveryChannelPolicies.FileNone,
+        });
+        dbContext.DefaultDeliveryChannels.Add(new DLCS.Model.DeliveryChannels.DefaultDeliveryChannel()
+        {
+            Customer = customerId,
+            MediaType = "application/*",
+            DeliveryChannelPolicyId = KnownDeliveryChannelPolicies.None,
+        });
+
+        await dbContext.SaveChangesAsync();
+        
+        A.CallTo(() =>
```

Comment:
This shouldn't be required for this test to pass, should never make it to the point where ti calls engine.